### PR TITLE
adding changlog for 3.8.2

### PR DESCRIPTION
**Pull request by picodoth**
screen shot 2018-08-18 at 11 42 26 am

有几个 demo 和文档的 commits 没加到 changelog

potential merging:
https://github.com/ant-design/ant-design/pull/11761
https://github.com/ant-design/ant-design/pull/11767

### CODE REVIEW

**Comment by afc163** [comment]
- [ ] https://github.com/ant-design/ant-design/pull/11702

**Comment by afc163** [comment]
master ci 挂了。

**Comment by afc163** [comment]
👍，写的挺好。

File: CHANGELOG.zh-CN.md [hunk 1]
```diff
@@ -15,6 +15,20 @@ timeline: true
...
 
+## 3.8.2
+
+`2018-8-18`
+- 🐞 修复 Tag 组件 `visible` 属性初始值为 false 时仍然可见的问题。[#11757](https://github.com/ant-design/ant-design/issues/11757)
+- 🐞 修复 Modal 文本在双击打开的时候会被选中的问题。[#11777](https://github.com/ant-design/ant-design/issues/11777)
+- 🐞 修复 Rate 组件点击时的样式问题。[#11736](https://github.com/ant-design/ant-design/issues/11736)
+- 🐞 修复 Badge 组件在子元素为 `display:block` 时的样式问题。[#84119d8](https://github.com/ant-design/ant-design/commit/84119d8959d55edf535a9cac5ff532e61b6ee698)
+- Drawer
+  - 🐞 修复 Drawer 组件使用的 `pointer-events` 在 IE10 不支持的问题。[#11583](https://github.com/ant-design/ant-design/issues/11583)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: 应该是 https://github.com/ant-design/ant-design/pull/11702 这个问题。

Comment (afc163) [reply CHANGELOG.zh-CN.md]: 不应该写 `pointer-events`，这个太细了，应该写修复这个问题：

image

**Comment by picodoth** [comment]
@afc163 合了？

**Pull request merged**